video/out/w32_common.c

```diff
@@ -1526,6 +1542,26 @@ static LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam,
             ((LPNCCALCSIZE_PARAMS) lParam)->rgrc[0].top -= get_title_bar_height(w32);
         }
         break;
+    case WM_IME_STARTCOMPOSITION: {
+        HIMC imc = ImmGetContext(w32->window);
+        COMPOSITIONFORM cf = {.dwStyle = CFS_POINT, .ptCurrentPos = {0, 0}};
+        ImmSetCompositionWindow(imc, &cf);
```

Comment:
maybe something like this, to be concise?
```suggestion

        ImmSetCompositionWindow(imc, &(COMPOSITIONFORM){ .dwStyle = CFS_POINT, .ptCurrentPos = {0, 0} });
```